# Generated by Django 4.0.4 on 2022-05-10 10:26

import django.core.validators
from django.db import migrations, models


class Migration(migrations.Migration):

    initial = True

    dependencies = [
    ]

    operations = [
        migrations.CreateModel(
            name='GeoObject',
            fields=[
                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('name', models.CharField(max_length=64, verbose_name='Название')),
                ('slug', models.CharField(max_length=64, unique=True, verbose_name='Слаг')),
                ('gismeteo_id', models.IntegerField(blank=True, null=True, unique=True, verbose_name='ID географического объекта (Gismeteo)')),
                ('latitude', models.FloatField(blank=True, null=True, validators=[django.core.validators.MaxValueValidator(90, message='Широта не может быть больше 90'), django.core.validators.MinValueValidator(-90, message='Широта не может быть меньше -90')], verbose_name='Широта расположения географического объекта')),
                ('longitude', models.FloatField(blank=True, null=True, validators=[django.core.validators.MaxValueValidator(180, message='Долгота не может быть больше 180'), django.core.validators.MinValueValidator(-180, message='Долгота не может быть меньше -180')], verbose_name='Долгота расположения географического объекта')),
            ],
            options={
                'verbose_name': 'Географический объект',
                'verbose_name_plural': 'Географические объекты',
                'ordering': ('name',),
            },
        ),
    ]
